datalad/distribution/tests/test_create_sibling.py

```diff
@@ -37,6 +37,7 @@
 )
 from datalad.support.gitrepo import GitRepo
 from datalad.support.network import urlquote
+from ...tests.utils_testdatasets import _mk_submodule_annex
```

Comment:
why is this import relative, shouldn't it be absolute, or am I missing something? It is also absolute in ``test_get.py`` further down

Reply:
Done